framework/auth/oauth_scopes.py

```diff
@@ -62,9 +72,9 @@ class ComposedScopes(object):
                     (CoreScopes.NODE_BASE_WRITE, CoreScopes.NODE_CHILDREN_WRITE, CoreScopes.NODE_LINKS_WRITE)
 
     # Privileges relating to editing content uploaded under that node # TODO: Add wiki etc when implemented
-    NODE_DATA_READ = (CoreScopes.NODE_FILE_READ,)
+    NODE_DATA_READ = (CoreScopes.NODE_FILE_READ, CoreScopes.NODE_COMMENTS_READ)
```

Comment:
Scopes are on top of (but separate from) user permissions. They don't have to map to what the UI does (and oftentimes should not). Rather, it helps to think of them as bundles of what permissions you think a user would logically want to give out to a third party at the same time. ("site x can see my project titles and read comments I made, but they're not allowed to upload files")

You make a good case for this to be part of basic project metadata, so
`METADATA_READ` and `METADATA_WRITE` seem better homes than the `NODE_DATA_` grouping- the latter relates to file upload etc.
Just note that scopes